### DISCUSSION TITLE
[Table] Add more cell type option to support more field formats

### DISCUSSION THREAD

**Discussion by xiaobeiyang** [grafana/grafana#64369] [Dashboards, Panels & Visualizations]
Sometimes we need display different format in one column. For example, a text column value may be JSON or XML or YAML. It will be very helpful if the value is well formatted in Inspect value widget.
![image](https://user-images.githubusercontent.com/48786770/223654884-0c21a475-b8f7-460c-9c5b-f5881ce1646b.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82798. Feel free to continue the discussion around this there. Thank you!
